bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/gson/IntesisHomeJSonDTO.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.intesis.internal.gson;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * {@link IntesisHomeJSonDTO} is used for the JSon/GSon mapping
+ *
+ * @author Hans-Jörg Merk - Initial contribution
+ */
+public class IntesisHomeJSonDTO {
+
+    public static class response {
```

Comment:
Classes should start with uppercase character:
```suggestion
    public static class Response {
```

Reply:
Changed